.reaction/docker/scripts/.gitignore

```diff
@@ -0,0 +1 @@
+/entrypoint-PROD-build-with-src.sh
```

Comment:
Sorry about that, I wasn't aware that `.gitignore` files are not to be committed. I removed it and am ignoring my custom entrypoint script with `.git/info/exclude` now.